apps/lfx-one/src/app/modules/badges/badges-dashboard/badges-dashboard.component.scss

```
@@ -0,0 +1,52 @@
+    rounded-md border border-gray-300 text-xs font-medium text-gray-700
+    hover:bg-gray-50 transition-colors bg-white cursor-pointer;
+}
+
+.badge-filter-dot {
+  @apply absolute -top-1 -right-1 w-2 h-2 rounded-full bg-blue-500;
+}
+
+.badge-card-link {
+  @apply block w-full no-underline text-inherit cursor-pointer;
+
+  &:hover ::ng-deep .p-card {
```

Stylelint (17.6.0)

[error] Unexpected unknown pseudo-element selector "::ng-deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)